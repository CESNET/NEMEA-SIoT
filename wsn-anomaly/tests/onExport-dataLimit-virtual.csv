uint64 DEV_ADDR,time TIMESTAMP,double VALUE
11745387828182253569,2018-04-03T21:31:44.000,6.992797
11745387828182253569,2018-04-03T21:31:49.000,6.603814
11745387828182253569,2018-04-03T21:31:54.000,6.518628
11745387828182253569,2018-04-03T21:31:59.000,6.996523
11745387828182253569,2018-04-03T21:32:04.000,0.014719
11745387828182253569,2018-04-03T21:32:09.000,2.385228
11745387828182253569,2018-04-03T21:32:14.000,0.031222
11745387828182253569,2018-04-03T21:32:19.000,2.167598
11745387828182253569,2018-04-03T21:32:24.000,5.248788
11745387828182253569,2018-04-03T21:32:29.000,5.464208
11745387828182253569,2018-04-03T21:32:34.000,20.268659
11745387828182253569,2018-04-03T21:32:39.000,18.822299
11745387828182253569,2018-04-03T21:32:44.000,18.224317
11745387828182253569,2018-04-03T21:32:49.000,15.678061
11745387828182253569,2018-04-03T21:32:54.000,15.897764
11745387828182253569,2018-04-03T21:32:59.000,60.945228
