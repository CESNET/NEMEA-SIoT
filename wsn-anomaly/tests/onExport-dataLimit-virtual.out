11745387828182253569,1522791144.000000,3.788785,2.901781,"Lower hard limit","moving_variance","VALUE"
11745387828182253569,1522791174.000000,25.803411,2.901781,"Higher hard limit","moving_variance","VALUE"
11745387828182253569,1522791179.000000,12.921496,6.518628,"Higher hard limit","moving_median","VALUE"
11745387828182253569,1522791184.000000,13.584772,6.518628,"Higher hard limit","moving_median","VALUE"
11745387828182253569,1522791184.000000,26.223854,2.901781,"Higher hard limit","moving_variance","VALUE"
11745387828182253569,1522791189.000000,12.921496,6.518628,"Higher hard limit","moving_median","VALUE"
11745387828182253569,1522791189.000000,36.037678,2.901781,"Higher hard limit","moving_variance","VALUE"
11745387828182253569,1522791194.000000,12.921496,6.518628,"Higher hard limit","moving_median","VALUE"
11745387828182253569,1522791194.000000,28.430765,2.901781,"Higher soft limit","moving_variance","VALUE"
11745387828182253569,1522791194.000000,28.430765,2.901781,"Higher hard limit","moving_variance","VALUE"
11745387828182253569,1522791199.000000,12.921496,6.518628,"Higher hard limit","moving_median","VALUE"
11745387828182253569,1522791199.000000,21.784431,2.901781,"Higher soft limit","moving_variance","VALUE"
11745387828182253569,1522791204.000000,12.466414,6.518628,"Higher soft limit","moving_median","VALUE"
11745387828182253569,1522791204.000000,12.466414,6.518628,"Higher hard limit","moving_median","VALUE"
11745387828182253569,1522791204.000000,21.031535,2.901781,"Higher soft limit","moving_variance","VALUE"
11745387828182253569,1522791209.000000,10.658162,6.518628,"Higher soft limit","moving_median","VALUE"
11745387828182253569,1522791209.000000,18.328695,2.901781,"Higher soft limit","moving_variance","VALUE"
11745387828182253569,1522791214.000000,10.658162,6.518628,"Higher soft limit","moving_median","VALUE"
11745387828182253569,1522791214.000000,20.349419,2.901781,"Higher soft limit","moving_variance","VALUE"
11745387828182253569,1522791219.000000,23.108840,2.901781,"Higher soft limit","moving_variance","VALUE"
11745387828182253569,1522791224.000000,19.135343,2.901781,"Higher soft limit","moving_variance","VALUE"
11745387828182253569,1522791329.000000,4.813933,2.901781,"Lower hard limit","moving_variance","VALUE"
11745387828182253569,1522791334.000000,3.340707,2.901781,"Lower hard limit","moving_variance","VALUE"
11745387828182253569,1522791339.000000,1.449690,2.901781,"Lower hard limit","moving_variance","VALUE"
11745387828182253569,1522791344.000000,2.718344,2.901781,"Lower hard limit","moving_variance","VALUE"
11745387828182253569,1522791349.000000,2.862515,2.901781,"Lower hard limit","moving_variance","VALUE"
11745387828182253569,1522791354.000000,3.028274,6.518628,"Lower soft limit","moving_median","VALUE"
11745387828182253569,1522791354.000000,4.918185,2.901781,"Lower soft limit","moving_variance","VALUE"
11745387828182253569,1522791354.000000,4.918185,2.901781,"Lower hard limit","moving_variance","VALUE"
11745387828182253569,1522791359.000000,6.509292,2.901781,"Lower soft limit","moving_variance","VALUE"
11745387828182253569,1522791364.000000,7.299333,2.901781,"Lower soft limit","moving_variance","VALUE"
